src/components/checks/[share_token]/RenderExamQuestion.tsx

```
@@ -44,6 +45,40 @@ export default function RenderExamQuestion() {
+function QuestionHeader({ title, index, variant = 'inline' }: { title: string; index: number; variant?: 'inline' | 'absolute' | 'inline-left' }) {
+  if (variant === 'inline-left') return HeaderLeftInline({ title, index })
+  if (variant === 'absolute') return HeaderAbsolute({ title, index })
+
+  return HeaderInline({ title, index })
+}
+
+function HeaderLeftInline({ title, index }: { title: string; index: number }) {
+  return (
+    <div className='-mt-4 -ml-4 grid grid-cols-[auto_1fr] gap-3'>
+      <div className='ring-ring dark:ring-ring flex max-h-10 items-center rounded-tl-md rounded-br-md bg-neutral-300 px-3.5 py-1.5 ring-1 dark:bg-neutral-600/60'>{index + 1}</div>
+      <p className='mt-[0.4rem] text-lg font-semibold text-neutral-800 dark:text-neutral-200' children={title} />
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)